code/player/Player.RPCs.cs

```diff
@@ -0,0 +1,19 @@
+using Sandbox;
+
+using TTTReborn.Roles;
+
+namespace TTTReborn.Player
+{
+    partial class TTTPlayer
+    {
+        /// <summary>
+        /// Called by the server, updates client's `Role`.
```

Comment:
Very nit picky...

"Must be called on the server, updates TTTPlayer's `Role`.

Reply:
Hmm. Good idea regarding the missing separation between Client and Server methods.